ng-gridify.component.ts

```diff
@@ -89,6 +89,13 @@ export class ngGridifyComponent {
     this.currentPage = page;
   }
 
```

Comment:
Something like this would do it?? Unless you can think of a better way of course? :)

DeepValue(obj, key) {
    let c = obj; 
    try {
      key.split('.').forEach((p) => c = c[p]); 
      return c;
    }
    catch (error) {
      return obj[key];
    }
    
  }